Bottom: fb15f6ad7d831ba03a57691df288b0c6e0c6d243
Top:    32a1030b25ffb3fd0fa176575fc97effd40c9de0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-19 13:53:27 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
